content/en/docs/faas/lambda-manual-instrument.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Lambda Manual Instrumentation
+linkTitle: Lambda Manual Instrumentation
+weight: 11
+description:
+  Manually instrument your Lambdas with OpenTelemetry 
+---
+
+For languages not covered in the Lambda auto-instrumentation document, the community does not have a standalone instrumentation layer.
+
+Users will need to follow the generic instrumentation guidance for their chosen language and add the Collector Lambda layer to submit their data.
+
+### Instrument the Lambda with OTel
+
+Review the [language instrumentation guidance](https://opentelemetry.io/docs/instrumentation/) on how to manually instrument your application.
+
+### Add the ARN of the OTel Collector Lambda layer
+
+Once you've instrumented your application you should add the Collector Lambda layer to collect and submit your data to your chosen backend.
+
+Note: Lambda layers are a regionalized resource, meaning that they can only be used in the Region in which they are published. Make sure to use the layer in the same region as your Lambda functions.
+
+Find the supported regions and amd64(x86_64)/arm64 layer ARN in the table below for the ARNs to consume.
+
+### Configure the OTel Collector
+
+The configuration of the OTel Collector Lambda layer follows the OpenTelemetry standard.
```

Comment:
I would suggest to create a new file just for discussing the embeeded Otel collector and its configuration (how to configure, what are the supported components etc). Reason for that is because the embedded collector configuration would be the same for both manual and auto-instrumentation.

Reply:
added a separate collector doc draft